### DISCUSSION TITLE
part-ii/2018-87/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#232] [Question comments]
Part II 2018 Logic and Set Theory: Paper 3, Section II, G 

https://questions.tripos.org/part-ii/2018-87/

**Comment by artie2000** [comment]
<details>
<summary>(i)</summary>

No (bookwork).

</details>
<details>
<summary>(ii)</summary>

Yes: add the axioms "$\forall x:(x\neq e\implies x^n\neq e)$".

</details>
<details>
<summary>(iii)</summary>

Yes (this is straightforward).

</details>
<details>
<summary>(iv)</summary>

No. Add constants $a_1,a_2,\dots$ and axioms making the $\{a_i\}$ distinct and $a_1\geq\dots\geq a_k$. Then a model of these axioms is not well-ordered, yet $\N$ models every finite subset.

</details>
<details>
<summary>Last bit:</summary>

Yes. Observe that $T=\{t_1,\dots,t_n\}$ is equivalent to $t\coloneqq\bigwedge_i t_i$, and apply compactness to $S\models t$ to get that
$$F\models t\models S$$
for some finite $F\subseteq S$.

</details>

**Comment by dropdbassett** [comment]
<details><summary>An alternative solution to (iv)</summary>

If we could axiomatise well-orders, we could axiomatise finite well-orderings as follows: define "$x$ is a limit" as an abbreviation for $\forall y, (y < x) \Rightarrow (\exists z, (y < z \wedge z < x))$. Then adding the axiom $\neg (\exists x, \exists y, (x \ne y) \wedge x$ is a limit $\wedge y$ is a limit to our axiomatisation of well-orderings gives an axiomatisation of finite well-orderings.

Now a similar argument to (i) shows that we cannot axiomatise finite well-orderings.
</details>
